src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -885,7 +885,7 @@ public unsafe void CopyTo(Array array, int index)
                         }
                     }
                 }
-                else if (Ssse3.IsSupported && ((uint)m_length >= Vector512<byte>.Count * 2u))
+                else if (Ssse3.IsSupported && ((uint)m_length >= Vector128<byte>.Count * 2u))
```

Comment:
I think while you're on it, we can also replace various Sse2.And with just plain operators, e.g.
```cs
Vector128<byte> extractedLower = Sse2.And(shuffledLower, bitMask128);
```
to
```cs
Vector128<byte> extractedLower = shuffledLower & bitMask128;
```
but that's optional